docs/plugins/find_and_install_plugin.md

```diff
@@ -30,3 +30,13 @@ directly from within napari:
    ![napari viewer's Plugin dialog. At the bottom of the dialog, there is a place to install by name, URL, or dropping in a file.](/images/plugin-install-dialog.png)
 
 3. Click the “Install” button next to the input bar.
+
+```{note}
+When wanting to install a plugin from a URL you need to respect a specific syntax required by pypi!
+You need to add the vcs (version control system) information before the URL for pip to be able to handle the URL.
+You will find more information about this on the [pip documentation](https://pip.pypa.io/en/stable/topics/vcs-support/)
```

Comment:
Can I suggest rephrasing this?
```suggestion
If you want to install a plugin from a URL, you need to use the required syntax (see the the [pip documentation](https://pip.pypa.io/en/stable/topics/vcs-support/) for more information.)
```

Reply:
Even better :)

Reply:
I'm just a bit wary of adding details about *every* little thing in our guide, risking to dilute too much the information. But I don't feel strongly about this, so I'll defer to others!

Reply:
I tend to agree with @brisvag here.  I think what I would probably do in this case is to modify the second bullet point as follows:

```markdown
2. In the resulting window that opens, where it says “Install by name/URL”, enter the
    name of the plugin you want to install (or any valid pip
    [requirement specifier](https://pip.pypa.io/en/stable/reference/requirement-specifiers/) or
    [VCS scheme](https://pip.pypa.io/en/stable/topics/vcs-support))
```

which renders as:

In the resulting window that opens, where it says “Install by name/URL”, enter thename of the plugin you want to install (or any valid pip [requirement specifier](https://pip.pypa.io/en/stable/reference/requirement-specifiers/) or [VCS scheme](https://pip.pypa.io/en/stable/topics/vcs-support))

and leave it to the pip documentation to explain what valid arguments to pip install are

Reply:
As in, you don't think the pip docs make that clear?